src/styles/blocks/header.scss

```diff
@@ -38,5 +38,11 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+
+    transition: color 0.3s linear;
```

Comment:
Maybe create some main values as mentioned in checklist p4.

Reply:
It's not used frequently, that's why I haven't created a variable, but who knows